docs/detections/api/rules/rules-api-export.asciidoc

```diff
@@ -1,14 +1,18 @@
 [[rules-api-export]]
 === Export rules
 
-Exports rules to an `.ndjson` file. Please note that you cannot export prebuilt rules, but they are available at https://github.com/elastic/detection-rules/tree/main/rules/.
+Exports rules to an `.ndjson` file. The following configuration items are also included in the `.ndjson` file:
 
-NOTE: Console supports {es} APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
+* Actions
+* Exception lists
+
```

Comment:
++

Reply:
OK, I'll work on adding the example to the page. In the meantime, @yctercero could you take a quick look at that example on #1170 and make sure it's all good for public consumption? Just want to make sure there isn't any internal or proprietary info that we should mock out for a sample.

Reply:
@yctercero @benironside I added a [**Response payload**](https://security-docs_1434.docs-preview.app.elstc.co/guide/en/security/master/rules-api-export.html#_response_payload_9) section and included the sample file contents. Please take a look and let me know what you think.

I think it seems a bit long, and I wonder about keeping it up to date. We'll need to update it any time the schema for the rule export file changes, but we don't always have good visibility into those kinds of changes. But if the usefulness of including the example outweighs those concerns, that's fine too.

Reply:
I'm going back and forth on this :) Do you think something like this would do the trick?
```ndjson
{"id":"d4db8800-30df-11ec-88a5-fb21b48c9b4e","rule_id":"query-with-single-exception-list"[....]} // Rule
{"id":"cd62f410-30de-11ec-88a5-fb21b48c9b4e","list_id":"simple_list"[...]} // Exception list parent container
{"id":"e54ffbe0-30de-11ec-88a5-fb21b48c9b4e","item_id":"my-exception-item","list_id":"simple_list"[...]} // Exception list item
{"exported_rules_count":1,"missing_rules":[],"missing_rules_count":0,"exported_exception_list_count":1,"exported_exception_list_item_count":1,"missing_exception_list_item_count":0,"missing_exception_list_items":[],"missing_exception_lists":[],"missing_exception_lists_count":0} // Export summary
```

So the focus isn't on the schema of the rule or exceptions, but just give them an idea that each line in the file represents an object (either rule, exception list parent container or exception list item) and the last line includes a summary of what was exported.

Reply:
Thanks @yctercero! Yes, a shorter version is more digestible, and the `//` annotations seem very helpful. I also like how you described the file contents, so stole that too. 😄 Preview has been updated.